core/tests/protractor_utils/AdminPage.js

```diff
@@ -131,19 +134,22 @@ var AdminPage = function() {
     await action.click('Admin roles tab button', adminRolesTab);
     await waitFor.pageToFullyLoad();
 
-    await expect(adminRolesTab.getAttribute('class')).toMatch('active');
+    expect(await adminRolesTab.getAttribute('class')).toMatch('active');
     await waitFor.visibilityOf(
       adminRolesTabContainer, 'Roles tab page is not visible.');
   };
 
   var saveConfigProperty = async function(
       configProperty, propertyName, objectType, editingInstructions) {
+    await waitFor.visibilityOf(
+      configProperty.element(
+        by.css('.protractor-test-config-title')));
     var title = await configProperty.element(
       by.css('.protractor-test-config-title')).getText();
     if (title.match(propertyName)) {
       await editingInstructions(
         await forms.getEditor(objectType)(configProperty));
-      await saveAllConfigs.click();
+      await action.click('Saves All Configs', saveAllConfigs);
```

Comment:
`action.click` interprets the first argument as the name of the element, not as a description of what it does

```suggestion
      await action.click('Save All Configs Button', saveAllConfigs);
```